macOS/DuckDuckGo/Preferences/View/PreferencesRootView.swift

```diff
@@ -32,7 +32,7 @@ enum Preferences {
         static var sidebarWidth: CGFloat {
             switch Locale.current.languageCode {
             case "en":
-                return 310
+                return 315
```

Comment:
This is required to avoid PIR sidebar item title fold into two lines. I will raise it during design review.